docs/tutorials/login-metamask.md

```diff
@@ -3,7 +3,202 @@ sidebar_position: 3
 description: It covers the providers integration, finishing with the login and logout with your Metamask wallet provider.
 ---
 
-# Authentication with Metamask
+import MetamaskApp from '@site/src/components/metamask'
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
-To be defined
+# Login with Metamask
+In this tutorial you will learn how you can integrate Metamask provider from the Catalog in your dapp and implement operations like login and logout
 
+## Requirements
+For the tutorial you will need:
+
+- A extension of [Metamask](https://metamask.io/) installed in the browser
+- [node](https://nodejs.org/en/) and [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) need to be installed
+
+## How to install Catalog Provider
+Firstable to install [Catalog Provider](../catalog/providers/README.md) you just need to run:
+
+<Tabs>
+  <TabItem label="yarn" value="yarn" default>
+
+```bash
+yarn add @nevermined-io/catalog-providers
+```
+
+  </TabItem>
+  <TabItem label="npm" value="npm">
+
+```bash
+npm install --save @nevermined-io/catalog-providers
+```
+  </TabItem>
+</Tabs>
+
+## Config your network (optional)
```

Comment:
I think this section should come at the end and use only mumbai and matic as an example